model/websocket_message.go

```diff
@@ -15,6 +15,7 @@ const (
 	WEBSOCKET_EVENT_POST_DELETED        = "post_deleted"
 	WEBSOCKET_EVENT_CHANNEL_DELETED     = "channel_deleted"
 	WEBSOCKET_EVENT_CHANNEL_CREATED     = "channel_created"
+	WEBSOCKET_EVENT_CHANNEL_UPDATED     = "channel_updated"
```

Comment:
can you also help submit a PR that adds the new WebSocket event to our API documentation: https://github.com/mattermost/mattermost-api-reference/blob/master/v4/source/introduction.yaml#L192